weave-js/src/components/Panel2/ChildPanel.tsx

```diff
@@ -575,6 +577,7 @@ export const ChildPanel: React.FC<ChildPanelProps> = props => {
   }, [selectedPath]);
 
   const isPanelSelected =
+    selectedDocumentId === documentId &&
```

Comment:
is this true in the board view as well?

Reply:
yep!

https://github.com/wandb/weave/assets/17016170/4339466c-bd4a-44df-8ad3-3061fac81a7f

Reply:
just fyi - in boards where the root panel isn't assigned a doc id, the string "input" is assigned as the default ID per [this line](https://github.com/wandb/weave/blob/118d337d7663087ea5de42c9098ac4233ea2eb20/weave-js/src/components/Panel2/PanelPanel.tsx#L318): `... ?? weave.expToString(props.input)`

that was just the behavior from before we introduced explicit doc ids